Uchu.StandardScripts/NexusTower/MaelstromConverter.cs

```diff
@@ -0,0 +1,32 @@
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.NexusTower
+{
+    [ScriptName("ScriptComponent_1537_script_name__removed")]
+    public class MaelstromConverter : ObjectScript
+    {
+        /// <summary>
+        /// Creates the object script.
+        /// </summary>
+        /// <param name="gameObject">Game object to control with the script.</param>
+        public MaelstromConverter(GameObject gameObject) : base(gameObject)
+        {
+            Listen(gameObject.OnInteract, player =>
+            {
+                // Let the player trade 25 Maelstrom Infected Bricks for 5 faction tokens
+                var inventory = player.GetComponent<InventoryManagerComponent>();
+                inventory.RemoveLotAsync(Lot.MaelstromInfectedBrick, 25);
```

Comment:
maybe add a check here if the player has 25 bricks?

Reply:
The client doesn't let the player interact if they have less than 25 bricks, but you're right it would probably be better to check it here as well